beacon-chain/db/deposits.go

```diff
@@ -22,19 +23,20 @@ var (
 
 // InsertDeposit into the database. If deposit or block number are nil
 // then this method does nothing.
-func (db *BeaconDB) InsertDeposit(ctx context.Context, d *pb.Deposit, blockNum *big.Int) {
+func (db *BeaconDB) InsertDeposit(ctx context.Context, d *pb.Deposit, blockNum *big.Int, depositRoot [32]byte) {
```

Comment:
why not sort it in insertion by blockNum ?

Reply:
added sorting upon insertion